packages/api/src/hooks/useAvailableAccounts.ts

```diff
@@ -0,0 +1,31 @@
+import { useMemo } from 'react';
+import useAccountTypes from './useAccountTypes';
+import useLandingCompany from './useLandingCompany';
+
+/** A custom hook to get available accounts for every landing companies */
+const useAvailableAccounts = () => {
+    const { data: landing_company_data } = useLandingCompany();
+    const { data: financial_accounts } = useAccountTypes(landing_company_data?.financial_company);
+    const { data: gaming_accounts } = useAccountTypes(landing_company_data?.gaming_company);
+    const { data: virtual_accounts } = useAccountTypes(landing_company_data?.virtual_company);
+
+    const available_accounts = useMemo(() => {
+        if (!financial_accounts && !gaming_accounts && !virtual_accounts) return;
+
+        return {
+            /** List of available financial accounts */
+            financial_accounts,
```

Comment:
@farhan-nurzi-deriv please add landing_company to each of these as well

Reply:
<img width="344" alt="Screenshot 2023-08-25 at 12 14 20 PM" src="https://github.com/binary-com/deriv-app/assets/125247833/f2148658-78d9-4c83-81d8-fb8fc398e2df">